assets/scss/layouts/header/_header.scss

```diff
@@ -132,6 +131,16 @@
     }
 }
 
+.header-logo-position-left {
+    float: left;
+    margin-left: remCalc(40px);
+}
+
+.header-logo-position-right {
+    float: right;
+    margin-right: remCalc(40px);
+}
+
```

Comment:
can't put it on `header-logo`, can't predict how the user will make the logo to look like with white padding etc...

![screen shot 2016-04-21 at 2 12 55 pm](https://cloud.githubusercontent.com/assets/8094145/14724464/30085b98-07cb-11e6-9c2f-8468efe3df78.png)

I am waiting for response from claire on the margin for mobile breakpoint